analysis/tests/utility_analysis_engine_test.py

```diff
@@ -346,19 +346,21 @@ def test_multi_parameters(self):
                                per_partition_error_max=-1.0,
                                expected_cross_partition_error=-0.5,
                                std_cross_partition_error=0.5,
-                               std_noise=11.6640625,
+                               std_noise=5.87109375,
```

Comment:
Is this change here because we are now using twice more budget?
Please could you add a comment mentioning the epsilon and delta parameters for which the STD was computed?

Reply:
Yes, that's because of having 2 times more budget. I checked that in Colab and it aligned (C++ implementation has relative precision 1e-3)